### PR TITLE
feat: migration script + some fixes

### PR DESCRIPTION
**Pull request by ebezzi**

### CODE REVIEW

File: scripts/cxg_admin.py [hunk 1]
```diff
@@ -299,6 +299,37 @@ def get_public_datasets(ctx):
     published_datasets = dataset_details.get_public_dataset_details()
     print(json.dumps(published_datasets, indent=2))
 
+@cli.command()
+@click.pass_context
+def migrate_redesign_read(ctx):
+    """
+    Print id, name, organism, tissue, assay, sex, cell_count, explorer_url, and S3 uris for all public datasets
```
Comment (nayib-jose-gloria) [inline scripts/cxg_admin.py:306]: can you update these command descriptions?

File: scripts/cxg_admin_scripts/migrate.py [hunk 2]
```diff
@@ -201,3 +205,330 @@ def backfill_processing_status_for_datasets(ctx):
...
+
+    # for dataset in dataset_versions:
+    #     if not dataset["status"]:
+    #         print("skipping")
+    #         print(dataset)
+    #         continue
+    #     DatasetStatus.from_json(json.dumps(dataset["status"]))
+
+    for dataset in dataset_versions:
+        if dataset["metadata"]["schema_version"] not in  ["3.0.0", "2.0.0"]:
```
Comment (nayib-jose-gloria) [inline scripts/cxg_admin_scripts/migrate.py:396]: do we still want 2.0.0 datasets?

File: scripts/cxg_admin_scripts/migrate.py [hunk 3]
```diff
@@ -201,3 +205,330 @@ def backfill_processing_status_for_datasets(ctx):
...
+        artifacts = json.load(f)
+
+    from backend.layers.common.entities import CollectionMetadata, DatasetStatus, DatasetMetadata
+
+    # for collection in collection_versions:
+    #     print(collection["metadata"])
+    #     CollectionMetadata.from_json(json.dumps(collection["metadata"]))
+
+    # for dataset in dataset_versions:
+    #     if not dataset["status"]:
```
Comment (nayib-jose-gloria) [inline scripts/cxg_admin_scripts/migrate.py:389]: Did we ever figure out why some datasets had no status set? 

Also, can we remove this commented out code if its been resolved?

File: scripts/cxg_admin_scripts/migrate.py (hunk 2 above)
Comment (ebezzi) [reply scripts/cxg_admin_scripts/migrate.py:396]: It's a good question. They're probably all tombstones, but there is no harm in migrating them though.

File: scripts/cxg_admin_scripts/migrate.py (hunk 3 above)
Comment (ebezzi) [reply scripts/cxg_admin_scripts/migrate.py:389]: Not yet, I'll look into them.

Comment (ebezzi) [reply scripts/cxg_admin_scripts/migrate.py]: They all belong to tombstoned collections.

File: scripts/cxg_admin_scripts/migrate.py (hunk 2 above)
Comment (ebezzi) [reply scripts/cxg_admin_scripts/migrate.py]: They should not exist in prod, so I wouldn't worry.

**Pull request merged**